plugins/videobasedtracker/Types.h

```diff
@@ -78,6 +75,18 @@ namespace vbtracker {
     typedef std::vector<LedGroup> LedGroupList;
     typedef std::vector<EstimatorPtr> EstimatorList;
     /// @}
+
+    /// Container for the information needed to define a sensor,
+    /// both the patterns of its beacons and their 3D spatial coordinates.
+    typedef struct {
+      PatternStringList patterns;
+      Point3Vector positions;
+      size_t requiredInliers;
+      size_t permittedOutliers;
+    } SensorDescription;
```

Comment:
should just be struct SensorDescription, the typedefs are only needed in c-compilable code